nifi-assembly/pom.xml

```diff
@@ -363,147 +363,6 @@ language governing permissions and limitations under the License. -->
         </dependency>
     </dependencies>
 
-    <properties>
```

Comment:
@JPercivall I moved them into the pom of nifi-resources so that the filtering could happen before the files are packaged.  This kept me from needing to duplicate them so that the toolkit could bundle a valid nifi.properties, instead I can just resolve an up to date one.